installer/go/README.md

```diff
@@ -0,0 +1,201 @@
+# FlowFuse Device Agent Installer
+
+A Go-based installer for the FlowFuse Device Agent that automatically sets up Node.js, installs the device agent package, and configures it as a system service.
+
+## Getting Started
+
+### Requirements
+
+- Linux, macOS, or Windows
+- Internet connection for downloading dependencies
+- Administrator/root privileges for system service installation
+
+### Installation
+
+Download the installer binary for your platform and run:
+
+```bash
+# Make the binary executable (Linux/macOS)
+chmod +x flowfuse-device-agent-installer
+
+# Install with one-time code from FlowFuse
+./flowfuse-device-agent-installer --otc YOUR_ONE_TIME_CODE
+```
```

Comment:
Sorry, I missed this in my review - this is linux/mac only. 

Also, it is missing the `--ff-url` param and so will not work as-is.

Windows instructions would need to inform the user to use an elevated command prompt and drop the `./`

Reply:
it may even need to mention unblocking the file: https://bin95.com/articles/safety/unblock-downloads.htm

Reply:
I consider this resolved with recent changes